mscore/musescore.cpp

```diff
@@ -7845,22 +7858,29 @@ void MuseScore::init(QStringList& argv)
             qApp->processEvents();
             }
 
-      if (!MScore::noGui)
+      // Best not to show this since the font used to display the message
+      // isn't updated till updateUiStyleAndTheme()
+      // showSplashMessage(sc, tr("Updating user interface and theme…"));
+      if (!MScore::noGui) {
             MuseScore::updateUiStyleAndTheme();
+            }
       else {
             genIcons(); // in GUI mode generated in updateUiStyleAndTheme()
             noSeq = true;
             }
 
       // Do not create sequencer and audio drivers if run with '-s'
       if (!noSeq) {
+            showSplashMessage(sc, tr("Creating sequencer and audio driver…"));
```

Comment:
I think “Creating” should be “Initializing”, since “creating” is a programming term that probably wouldn't make sense to most users.

Reply:
OK